README.md

```diff
@@ -1260,7 +1259,7 @@ Update Time, five active automations, webhooks.
   * [deSEC](https://desec.io) - Free DNS hosting with API support, designed with security in mind. Runs on open-source software and is supported by [SSE](https://www.securesystems.de/).
   * [dns.he.net](https://dns.he.net/) — Free DNS hosting service with Dynamic DNS Support
   * [Zonomi](https://zonomi.com/) — Free DNS hosting service with instant DNS propagation. Free plan: 1 DNS zone (domain name) with up to 10 DNS records.
-  * [dnspod.com](https://www.dnspod.com/) — Free DNS hosting.
+  * [Gcore](https://gcore.com/dns/) — Free DNS hosting with unlimited requests and domains free. DDoS protection/WAAP included
```

Comment:
they still offer CDN for free, why remove mention of that?

Reply:
no, you moved score here and removed mention of its quite significant CDN offer, I am wondering why

Reply:
yes that would be ideal

Reply:
you are welcome to enhance any existing entry - those checklists are more for new entries.